src/router/index.ts

```diff
@@ -263,6 +264,20 @@ const routes: Array<RouteRecordRaw> = [
           projectref: parseRef(firstEntry(route.params.projectref)),
         }),
       },
+      {
```

Comment:
Please read https://github.com/agola-io/agola-web/pull/89/files#r1418635633. Use different routes for update and create.